modules/distributor/forwarder/manager.go

```diff
@@ -18,8 +18,8 @@ import (
 )
 
 const (
-	defaultWorkerCount = 2
-	defaultQueueSize   = 100
+	defaultWorkerCount = 5
+	defaultQueueSize   = 10000
```

Comment:
i'm not comfortable with just a drastic increase in the default queue size here.

Reply:
I reduced to `1000` in [abe96d7](https://github.com/grafana/tempo/pull/3234/commits/abe96d7f892dfc8544e3de1615f8cfb0b8200ff7).